### PR TITLE
Comparison with blank crash

### PR DESCRIPTION
**Pull request by anderson-joyle**
Issue https://github.com/microsoft/Power-Fx/issues/1301.

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Core/IR/BinaryOpMatrix.cs [hunk 1]
```diff
@@ -271,9 +271,13 @@ private static BinaryOpKind GetBooleanBinaryOp(PowerFx.Syntax.BinaryOpNode node,
                 case DKind.ObjNull:
                     switch (node.Op)
                     {
-                        case BinaryOp.NotEqual:
+                        case BinaryOp.NotEqual:
+                        case BinaryOp.Less:
+                        case BinaryOp.Greater:
```
Comment (CarlosFigueiraMSFT) [inline src/libraries/Microsoft.PowerFx.Core/IR/BinaryOpMatrix.cs:276]: It feels wrong to use NeqNull for `<` and `>` (and EqNull for `>=` and `<=`). Should we introduce new op kinds?

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs [hunk 2]
```diff
@@ -349,6 +349,8 @@ private ValueTask<FormulaValue> VisitBinaryOpNode(BinaryOpNode node, EvalVisitor
                 case BinaryOpKind.EqDecimals:
                     return OperatorBinaryEq(this, context, node.IRContext, args);
                 case BinaryOpKind.EqNullUntyped:
+                case BinaryOpKind.GeqNull:
```
Comment (CarlosFigueiraMSFT) [inline src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs:352]: >case BinaryOpKind.GeqNull: [](http://example.com/codeflow?start=16&length=26)

OperatorBinaryGeq?

**Review by CarlosFigueiraMSFT — APPROVED** [review]
:shipit:

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**